src/ngraph/runtime/gpu/cuda_emitter.cpp

```diff
@@ -205,8 +205,7 @@ size_t runtime::gpu::CUDAEmitter::build_pad(const runtime::gpu::GPURuntimeContex
                                           0,
                                           NULL, // shared mem and stream
                                           args_list,
-                                          0));  // arguments
-            CUDA_SAFE_CALL(cuCtxSynchronize()); // Retrieve and print output.
+                                          0)); // arguments
```

Comment:
If you see lines like this you should revert them so they don't show up in the review. The only thing that changed on this like is number of spaces before the comment char.

Reply:
This is from clang format.